src/shell/dotnet/Shell/Shell.csproj

```diff
@@ -38,6 +38,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
+		<Folder Include="Preload\" />
```

Comment:
There's no `EmbeddedResource` item?

Reply:
https://github.com/morganstanley/ComposeUI/commit/7c30c502af23e0f3b578f53471f3ab9cbf3e95f7

Something went wrong with the history...
Let me recommit this